cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -569,3 +569,148 @@ Feature: JWT Authenticator - Fetch signing key
     """
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  Scenario: jku is unfollowed - security check
+    Given I initialize another JWKS endpoint with file "myFirstJWKs.json" with key number 1
```

Comment:
Why do we need key numbers? Could we use file name as a key?
I guess it will mage scenarios much more readable.

Reply:
I agree with @sashaCher

Reply:
Good comment
Done :)